exo-create/features/steps/steps-when.ls

```diff
@@ -10,7 +10,7 @@ require! {
 
 module.exports = ->
 
-  @When /^executing "([^"]*)"$/ (command, done) ->
+  @When /^executing "([^"]*)"$/, timeout: 20_000, (command, done) ->
```

Comment:
This step is called in `create-service.feature`

```
 When executing "exo-create service users-service test-author exoservice-es6-mongodb user manage users"
```

For whatever reason, it took about 10 seconds to create the service, so I just doubled that time. I'm not sure why it's taking so long though, because I'm pretty sure all we're doing is rendering a template.